package.json

```diff
@@ -22,11 +22,11 @@
     }
   ],
   "scripts": {
-    "size": "size-limit",
-    "test": "standard && ava",
```

Comment:
@suphalb this was accidentally removed, can you put this back?

Reply:
@f2prateek - can you please consider this latest commit ? https://github.com/segmentio/analytics-node/commit/b0b7874250340d87e37a58f24681076f11cd371e

Seems this time the CI build was successful with the small tweaks you suggested.